Dockerfile

```diff
@@ -27,41 +27,53 @@ ENV AIIDALAB_DEFAULT_GIT_BRANCH master
 ENV AIIDALAB_DEFAULT_APPS ""
 
 USER root
+WORKDIR /opt/
 
 # Install OS dependencies.
+# libssl-dev needed for ?
+# libffi-dev needed for ?
+# povray needed for structure editor widget
 RUN apt-get update && apt-get install -y  \
     ca-certificates       \
     file                  \
     libssl-dev            \
     libffi-dev            \
     povray                \
     python3-pip           \
-    python3-setuptools    \
-    python3-wheel         \
   && rm -rf /var/lib/apt/lists/*
 
-# Install what is needed for Jupyter Lab.
+# Dependencies needed for Jupyter Lab.
 RUN apt-get update && apt-get install -y \
      nodejs                \
      npm                   \
   && rm -rf /var/lib/apt/lists/*
 
-# Upgrade pip.
-RUN /usr/bin/python3 -m pip install -U pip
-
-# Install Jupyter-related things in the root environment.
-RUN /usr/bin/pip3 install          \
-    'jupyterhub==1.4.0'            \
-    'jupyterlab==3.0.14'            \
-    'notebook==6.3.0'
-
-# Install ngrok to be able to proxy AiiDA RESTful API server.
+# Install ngrok to be able to proxy AiiDA RESTful API server. ### is this used in the home app?
```

Comment:
is this currently needed by the "base"  aiidalab?

Reply:
@csadorf, can you please reply to this?

Reply:
Ngrok was added for in the context of an AiiDA tutorial to make it significantly easier to reverse-proxy the AiiDA web server. It is not used by the home app. I would recommend to keep it.

Reply:
Alright!